reactivesocket-cpp/src/Frame.h

```diff
@@ -8,15 +8,19 @@
 
 /// Needed for inline d'tors of frames.
 #include <folly/io/IOBuf.h>
+#include <folly/io/IOBufQueue.h>
+#include <folly/Memory.h>
```

Comment:
is this headed needer in the Frames.h header file?
Can you include it in the cpp file only?